optaplanner-docs/src/main/asciidoc/Chapter-Quick_start/Chapter-Quick_start.adoc

```diff
@@ -1,5 +1,7 @@
+
 [[quickStart]]
-= Quick Start
+= Getting Started: A Cloud Balancing Demonstration
+//Christy suggested that this isn't really a "quickstart". I don't know how you feel about renaming a chapter?
```

Comment:
Hey Gemma, can we put these sort of comments in the PR, but not in the actual source (so it doesn't get merged by accident)?

Can you explain why this isn't really a quickstart? We can rename it or restructure it if it's not correct, of course. I want to understand why/how. The goal of this chapter is for users to get started quickly by learning the minimal basics to implement an optaplanner case, without reading 350 pages of documentation (which they don't do).